docker-java-core/src/main/java/com/github/dockerjava/core/exec/StatsCmdExec.java

```diff
@@ -23,6 +25,10 @@ protected Void execute0(StatsCmd command, ResultCallback<Statistics> resultCallb
         WebTarget webTarget = getBaseResource().path("/containers/{id}/stats").resolveTemplate("id",
                 command.getContainerId());
 
+        if (isTrue(command.getNoStream())) {
+            webTarget = webTarget.queryParam("stream", "0");
```

Comment:
what 0 mean here?

Reply:
This is only equivalent to curls, such as 
curl https://172.17.42.1:2375/containers/example/stats?stream=0
curl https://172.17.42.1:2375/containers/example/stats?stream=false

Reply:
Also I saw parsing in https://github.com/moby/moby/blob/master/client/container_stats.go

Reply:
@alexey-atiskov please do not resolve conversations that were started by somebody else. It is up to the reviewer to decide whether something is resolved or not

Reply:
Oh, shame on me, I noticed this comment tool late(

Reply:
no problem :)